dfastmi/io/RiverParameterData.py

```diff
@@ -219,183 +226,175 @@ def collect_values1(
 
         Returns
         -------
-        data : List[List[float]]
+        data : int
```

Comment:
update documentation correctly, it is an int now, but the description says a list of lists

Reply:
Discussed, we will leave these comments for a later refactoring.
This will be tackled in https://issuetracker.deltares.nl/browse/DFAST-112